src/tests/CoreMangLib/system/delegate/miscellaneous/SecondLevelIndirectionWithNullThis.cs

```diff
@@ -0,0 +1,27 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+using System;
+using System.Reflection;
+using Xunit;
+
+public class Program
+{
+    [Fact]
+    public static int TestEntryPoint()
+    {
+        MethodInfo invokeMethod = typeof(Action).GetMethod("Invoke");
+        Action<Action> wrapperDelegate = invokeMethod.CreateDelegate<Action<Action>>();
+        Assert.NotNull(wrapperDelegate);
+
+        try
+        {
+            wrapperDelegate(null);
```

Comment:
For me, this hits the null reference exception in virtual dispatch at

```
coreclr!_CxxThrowException+0x97 [D:\a\_work\1\s\src\vctools\crt\vcruntime\src\eh\throw.cpp @ 82] 
coreclr!RealCOMPlusThrow+0x10b [D:\a\_work\1\s\src\coreclr\vm\excep.cpp @ 11176] 
coreclr!VSD_ResolveWorker+0x44d [D:\a\_work\1\s\src\coreclr\vm\virtualcallstub.cpp @ 1319] 
coreclr!ResolveWorkerAsmStub+0x6d [D:\a\_work\1\s\src\coreclr\vm\amd64\VirtualCallStubAMD64.asm @ 36] 
repro!Program.<Main>$+0xa3
```

Reply:
Well I didn't test this path to verify what it hits.
Reflection stack prevents null instance before reaching the stub. Maybe UnsafeAccessor?

Reply:
I've tested locally that `ldvirtftn` is calling and decided by this stub.